### PR TITLE
Product card

### PR DESCRIPTION
**Pull request by vsupruniuk**
- [DEMO LINK](https://vsupruniuk.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://vsupruniuk.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/blocks/rating.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  margin-bottom: 24px;
+  width: 16px;
+  height: 16px;
+}
+
+.card__rating-text {
+  font-size: 0.83rem;
+  margin-bottom: 24px;
+  color: #060b35;
+  font-weight: 400;
```
Comment (k0nstrukt-or) [inline src/blocks/rating.css:39]: це стандартне значення, можна і без нього

File: src/blocks/text.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.card__product-name {
+  display: flex;
+  margin-bottom: 4px;
+  font-size: 12px;
```
Comment (k0nstrukt-or) [inline src/blocks/text.css:4]: ти вже задав розмір шрифта

**Review by k0nstrukt-or — CHANGES_REQUESTED** [review]

File: src/blocks/rating.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+}
+
+.card__stars .star-5 {
+  background-image: url(../images/star.svg);
```
Comment (illiarerun) [inline src/blocks/rating.css:26]: all you need is to change `background-image` here

File: src/blocks/card.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  align-items: flex-start;
+  flex-grow: 0;
+  flex-shrink: 0;
```
Comment (illiarerun) [inline src/blocks/card.css:6]: redundant
```suggestion
```

File: src/blocks/buy-link.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
```
Comment (illiarerun) [inline src/blocks/buy-link.css:5]: redundant, block elemets will take up all of the available width
```suggestion
```

File: src/blocks/buy-link.css [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  font-weight: bold;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.card__buy:hover {
+  height: 38px;
```
Comment (illiarerun) [inline src/blocks/buy-link.css:18]: no, you should not change it here, use `box-sizing: border-box` so you wouldn't need to calculate it on hover
```suggestion
```

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,58 @@
...
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
```
Comment (jstmpelowycz) [inline src/index.html:11]: Format code properly

**Review by jstmpelowycz — APPROVED** [review]
Good job 🔥

**Pull request closed without merging**